doc/doc-requirements.txt

```diff
@@ -6,17 +6,7 @@ traitlets
 pandas
 ruamel.yaml
 git+https://github.com/pandas-dev/pandas-sphinx-theme.git@master
-# install BinderHub dependencies. We manually list them here because some
-# dependencies (like pycurl) can't be installed on ReadTheDocs and aren't
-# needed to build the docs.
-kubernetes==9.0.*
```

Comment:
How will these dependencies now get installed on read the docs?

Reply:
with `-e .`, setup.py installs `requirements.txt` https://github.com/jupyterhub/binderhub/blob/0b5080ade21dbda5ad159e675bacf25c61d8e422/setup.py#L50

Actually now I am thinking we even don't need here `-e/--editable` flag, only `.` should be enough. what do you think?